src/webgl/p5.Camera.js

```diff
@@ -2448,17 +2448,33 @@ p5.Camera = class Camera {
     rotatedCenter[1] += this.eyeY;
     rotatedCenter[2] += this.eyeZ;
 
+    // Compute new up vector to prevent flipping
+    let forward = createVector(
```

Comment:
The global `createVector` function isn't available in instance mode, can we import the vector class directly and do `new Vector(...)` here instead?

Reply:
Thank you for the review . I’ll refactor the code to avoid createVector while still keeping it clean and readable